charts/hush-sensor/values.yaml

```diff
@@ -18,6 +18,10 @@ namespace:
   create: true
   name: "hush-security"
 
+hushZone: ""
```

Comment:
Isn't this part of the token?

Reply:
What Rafi said. Also we probably don't want to expose this one in values.yaml if we don't have to since it's internal.

Reply:
Discussed with @horomena f2f.
Fixed the logic to use the mgmt zone in the token to get to the correct hush-security.com (sub)domain.